frontend/js/src/settings/import/ImportListens.tsx

```diff
@@ -49,47 +49,35 @@ export default function Import() {
         </div>
       )}
       <p>
-        Most users will want to import from Last.fm directly.
-        <br />
+        Import your existing listen history from other databases. To submit <em>new</em>
+        listens, please visit the{" "} <a href="https://listenbrainz.org/add-data/">Connect services</a>
+        and{" "} <a href="https://listenbrainz.org/settings/music-services/details/">Submitting data</a>
+        pages.
+      <br/>
         Fun Fact: The term <strong>scrobble</strong> is a trademarked term by
         Last.fm, and we cannot use it. Instead, we use the term{" "}
         <strong>listen</strong> for our data.
       </p>
 
-      <h3>Direct import from Last.fm and Libre.fm</h3>
+      <h3>Import from Last.fm and Libre.fm</h3>
       <p>
         The importer manually steps through your listen history and imports the
-        listens one page at a time. Should it fail for whatever reason, it is
-        safe to restart the import process. Running the import process multiple
-        times <strong>does not</strong> create duplicates in your ListenBrainz
-        listen history.
-      </p>
-      <p>
-        In order for this to work, you must disable the &#34;Hide recent
-        listening information&#34; setting in your Last.fm{" "}
-        <a href="https://www.last.fm/settings/privacy">Privacy Settings</a>.
-      </p>
-      <p>
-        Clicking the &quot;Import listens&quot; button will import your profile
-        now without the need to open LastFM.
-        <br />
-        You need to keep this page open for the tool to work, it might take a
-        while to complete. Though, you can continue doing your work. :)
+        listens one page at a time.
       </p>
+
+      <ul>
+        <li>In order for this to work, you must disable the &#34;Hide recent listening information&#34; setting in your Last.fm{" "} <a href="https://www.last.fm/settings/privacy" target="_blank">Privacy Settings</a>.</li>
```

Comment:
The [automated tests](https://github.com/metabrainz/listenbrainz-server/pull/2810/checks#step:4:1943) are complaining about this, apparently to make links safer for older browsers:

```suggestion
        <li>In order for this to work, you must disable the &#34;Hide recent listening information&#34; setting in your Last.fm{" "} <a href="https://www.last.fm/settings/privacy" target="_blank" rel="noreferrer">Privacy Settings</a>.</li>
```

Reply:
Thanks monkey - is it policy to have this on all external links?